library/core/src/main/java/com/google/android/exoplayer2/video/VideoRendererEventListener.java

```diff
@@ -94,8 +94,9 @@ void onVideoSizeChanged(int width, int height, int unappliedRotationDegrees,
    *
    * @param surface The {@link Surface} to which a first frame has been rendered, or {@code null} if
    *     the renderer renders to something that isn't a {@link Surface}.
+   * @param framePosition The position of the rendered first frame in milliseconds.
    */
-  void onRenderedFirstFrame(Surface surface);
+  void onRenderedFirstFrame(Surface surface, long framePosition);
```

Comment:
Please put `Ms` explicitly on the end of the variable name, for clarity. Ditto in a few places.